README.md

```diff
@@ -13,6 +13,34 @@ The __filter__ package is generally provided as a folder with source (`.java`) a
 
 A sample set of images are provided in an __"images"__ directory.
 
+## Creating an executable jar file
+If there is no automated build environment set up, the jar file can be created using "jar."
+1. Open up the CLI and change the directory to the projects root directory.
+2. Execute the command "javac Run.java" to compile the Run.java file.
+3. Execute the command "javac \*.java" in the package directories to compile/recompile the java files.
```

Comment:
What are "the package directories"?  If these instructions are so basic as to specify the "javac" command, they shouldn't sometimes presume greater sophistication.